README.md

```diff
@@ -18,7 +18,7 @@ The version of Apache Storm to install must match the one defined in the pom.xml
 
 Once Storm is installed, the easiest way to get started is to generate a brand new StormCrawler project using \: 
 
-`mvn archetype:generate -DarchetypeGroupId=com.digitalpebble.stormcrawler -DarchetypeArtifactId=storm-crawler-archetype -DarchetypeVersion=2.11`
+`mvn archetype:generate -DarchetypeGroupId=com.digitalpebble.stormcrawler -DarchetypeArtifactId=stormcrawler-archetype -DarchetypeVersion=2.11`
```

Comment:
archetype/pom.xml addresses this I think

Reply:
It is in the readme, so ` -DarchetypeGroupId=com.digitalpebble.stormcrawler ` needs to be adjusted otherwise using the new archetype won't work :D

Reply:
ah, I see (sorry I am being slow). I am wondering whether we should put 
`org.apache.stormcrawler.... 3.0-SNAPSHOT` there anyway. 
It means pppl will have to `mvn install` first but at least it will be consistent with the content of the repo
WDYT?

Reply:
Yeah we should update that README files regarding the archetype. It won't work at the moment anyway and will be fixed after a (hopefully soonish) 3.0.0 release.